src/main/resources/application.yaml

```diff
@@ -41,8 +41,8 @@ amqp:
 
 document:
   pdf:
-    hmctsImgKey: "hmcts2"
-    hmctsImgVal: "[userImage:hmcts.png]"
+    hmctsImgKey: hmcts2
+    hmctsImgVal: "[userImage:hmcts-2.png]"
```

Comment:
is this correct location? Does hmcts-2.png exist?

Reply:
yes, that's the new logo